src/webui/src/static/model/trialmanager.ts

```diff
@@ -48,6 +48,15 @@ class TrialManager {
     private latestMetricdataErrorMessage: string = ''; // metric-data-latest error message
     private isMetricdataRangeError: boolean = false; // metric-data-range api error filed
     private metricdataRangeErrorMessage: string = ''; // metric-data-latest error message
+    private metricsList: Array<any> = [];
+    private trialJobList: Array<any> = [];
+
+    public getMetricsList(): Array<any> {
+        return this.metricsList;
+    }
+    public getTrialJobList(): Array<any> {
```

Comment:
Trial job should have a type.

Reply:
had fixed. add type `MetricDataRecord TrialJobInfo`.